libkbfs/init.go

```diff
@@ -354,11 +354,18 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn, onI
 	go func() {
 		_ = <-interruptChan
 
+		close(done)
+
 		if onInterruptFn != nil {
 			onInterruptFn()
+
+			// Keep listening on the signal channel in case unmount fails the first
+			// time and user continues pressing Ctrl-C.
+			for range interruptChan {
```

Comment:
`interruptChan` is the channel that `os/signal` sends signals into, so we need to keep it open until program exits, i.e. not closing. I'll add comment in the code.